models/iaf_psc_exp.nestml

```diff
@@ -85,7 +85,7 @@ neuron iaf_psc_exp_neuron:
     shape I_shape_in = exp(-1/tau_syn_in*t)
     shape I_shape_ex = exp(-1/tau_syn_ex*t)
     function  I_syn mV = (curr_sum(I_shape_in, in_spikes) + curr_sum(I_shape_ex, ex_spikes) + I_e + currents)
-    V_abs' = -1/tau_m * V_abs + 1/C_m * I_syn
+    V_abs' = -V_abs/tau_m + (I_syn/C_m) * 1 [s**3 * A**2 /(kg * m**2)] * 1 [s/Gs]
```

Comment:
Yes, this one is complete nonsense. It is likely an aggregate of several fixes. This all equates to a factor of 1 nS. Fixed in update.